packages/next/build/index.ts

```diff
@@ -175,6 +176,12 @@ export default async function build(
       telemetry.record(events)
     )
 
+    if (config.eslint?.build) {
+      await nextBuildSpan
+        .traceChild('verify-eslint-setup')
+        .traceAsyncFn(() => verifyEslintSetup(dir, pagesDir, null))
```

Comment:
Do we want to block the build while this is running? We could move this to a separate worker to allow the build to continue and then exit with an error if linting fails

Reply:
Good call. Updated to use a new worker so it doesn't block build 👍